charts/weave-gitops/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: weave-gitops
+description: Weave Gitops helps you manage your flux installation
```

Comment:
This probably should be it's own PR no? Seems strange to lump this into the "Delete things PR"